### PR TITLE
IGAPP-429: Last update for inner categories

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/src/modules/categories/components/CategoryList.js [hunk 1]
```diff
@@ -54,6 +58,17 @@ const CategoryThumbnail = styled(Image)`
  * Displays a ContentList which is a list of categories, a caption and a thumbnail
  */
 class CategoryList extends React.Component<PropsType> {
+  getCachedThumbnail (category: CategoryModel): ?string {
```
Comment (klinzo) [inline native/src/modules/categories/components/CategoryList.js:61]: Why did you have to add this? is this used anywhere?

**Review by klinzo — COMMENTED** [review]
Hopefully we won't have to many merge conflicts with @maxammann s PR...

**Review by klinzo — APPROVED** [review]

File: native/src/modules/categories/components/CategoryList.js (hunk 1 above)
Comment (Taggotty) [reply native/src/modules/categories/components/CategoryList.js]: I think I just forgot to remove it again.

**Comment by Taggotty** [comment]
@klinzo @maxammann wouldn't it make sense to separate the search category list from the normal category list. First because it seams that they require different props and work differently. But mostly I think that this would generalize the search overall if we want to add something in the future.

**Comment by klinzo** [comment]
@Taggotty if you think that makes sense sure. Haven't looked into it myself, but your points seem reasonable :+1: Maybe we should do that in a separate PR after @maxammann merged his big refactoring PR though, what do you think?

**Review by maxammann — APPROVED** [review]
will merge asap

**Pull request merged**